libraries/AP_Baro/AP_Baro_DPS280.cpp

```diff
@@ -127,6 +133,17 @@ void AP_Baro_DPS280::set_config_registers(void)
     dev->write_register(DPS280_REG_PCONF, 0x54, true); // 32 Hz, 16x oversample
     dev->write_register(DPS280_REG_TCONF, 0x54 | calibration.temp_source, true); // 32 Hz, 16x oversample
     dev->write_register(DPS280_REG_MCONF, 0x07); // continuous temp and pressure.
+
+    if (is_dps310) {
+        // work around broken temperature handling on some sensors
+        // using undocumented register writes
+        // see https://github.com/infineon/DPS310-Pressure-Sensor/blob/dps310/src/DpsClass.cpp#L442
```

Comment:
```suggestion
        // see https://github.com/Infineon/DPS310-Pressure-Sensor/blob/3edb0e58dfd7691491ae8d7f6a86277b001ad93f/src/DpsClass.cpp#L442
```

Better off tagging against a commit, rather then just the branch, as these kinds of things have a tendency to get accidentally shifted as someone pushes to it.

Reply:
if they haven't tagged then the git hash can disappear as they update. The link will still likely take them to the right file.
It is hard to predict which will be more robust